README.md

```diff
@@ -1,17 +1,121 @@
-# Cluster Ingress Operator
+# OpenShift Ingress Operator
 
-Cluster Ingress Operator deploys and manages [HAProxy](http://www.haproxy.org) to provide highly available network ingress in [OpenShift](https://openshift.io/), enabling both OpenShift [Routes](https://docs.okd.io/latest/rest_api/apis-route.openshift.io/v1.Route.html) and Kubernetes [Ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/).
 
-The operator tries to be useful out of the box by creating a working default deployment based on the cluster's configuration.
+#### Ingress Operator enables Routes and Kubernetes Ingress on OpenShift.
```

Comment:
This heading seems superfluous and looks kind of strange (it renders in bold-face, so it looks like an emphatic statement rather than a heading).

Reply:
Intention is to have a one-sentence tagline/"what is this thing"... recommendations?

Reply:
"Ingress Operator enables external access to cluster services by configuring Ingress Controllers, which route traffic as specified by OpenShift [Route](https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html)  and Kubernetes [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources."

Reply:
Updated... better?